site/content/entry/yuzu-progress-report-oct-2022/index.md

```diff
@@ -0,0 +1,314 @@
+  >}}
+
+Switching subjects to smooth changes where you don’t even expect them, here’s a quality of life improvement byte[] added this month.
+In the past, writing a screenshot taken by yuzu to disk would noticeably block rendering the game.
+The user would perceive this as a relatively short freeze (the time of this freeze varies according to the size of the resolution multiplier in use).
+byte[] {{< gh-hovercard "9024" "made this fully asynchronous," >}} allowing for smooth gameplay even during intensive user photo sessions.
+
+## CPU, Kernel, services, and core fixes
+
+A fix for CoreTiming accuracy from a few months ago, as part of [Pull Request #8650](https://github.com/yuzu-emu/yuzu/pull/8650), was added to intentionally waste CPU cycles until a certain amount of time had passed, due to extremely poor support in Windows for high precision timing events. 
+But in Linux, no such fixes are needed, since Linux natively supports high-precision delays, so byte[] allowed Linux users to take advantage of this and {{< gh-hovercard "9040" "lower the high CPU usage." >}}
+Remember when we say that yuzu is faster on Linux? This is the kind of stuff we mean.
+
+`No Man's Sky` was released, and our tester [Law](https://github.com/Law022) found that it could boot with auto-stubbing enabled. 
+byte[] investigated the log file, and stubbed the previously unimplemented calls {{< gh-hovercard "9032" "CheckFriendListAvailability" >}} and {{< gh-hovercard "9033" "GetCacheStorageSize," >}} allowing `No Man's Sky` to boot without any stubbing.
+
+{{< imgs
+    "./nms.png| Still the best screensaver"
+  >}}
+
+`Bayonetta 3`'s release caused some issues that were tricky to track down. 
+Your resident [bunnei](https://github.com/bunnei) rabbit properly implemented `ListOpenContextStoredUsers`, and the related `StoreOpenContext` which is used to set stored opened user profiles.
+`ListOpenContextStoredUsers` was causing `Bayonetta 3` to crash on startup. 
+{{< gh-hovercard "9157" "Fixing the stubs" >}} allowed the game to finally boot, and while it is now playable, it is quite slow still. 
+Stay tuned for performance optimizations regarding this game!
+
+[german77](https://github.com/german77) took part in the fun, {{< gh-hovercard "9149" "stubbing the SetRecordVolumeMuted service," >}} allowing the game to boot.
+
+{{< imgs
+    "./bayo.png| Hot protagonists in your area (Bayonetta 3)"
+  >}}
+
+Another notable release from bunnei this month was the beginning of the {{< gh-hovercard "9071" "multi-process project." >}} 
+Since the very beginning, yuzu has only ever supported a single-process architecture, since games only ever would use one process on the Switch. 
+However, to help improve yuzu's accuracy to the Switch software, yuzu has been focusing on getting key operating system features working, and one of the most important is multi-process support, since this corresponds to how the Switch software actually works.
```

Comment:
```suggestion
Another notable release from bunnei this month was the start of the {{< gh-hovercard "9071" "multiprocess project." >}} 
Since the very beginning, yuzu has only ever supported a single-process architecture, as games only ever use one process on the Switch. 
However, to help improve yuzu's accuracy for Switch software, yuzu has been focusing on getting key operating system features working, and one of the most important is multiprocess support, since this corresponds to how Switch software actually works.
```